typescript/src/main.ts

```diff
@@ -262,6 +262,18 @@ function downloadTableAsCsv(tableId: string, separator = ",") {
   document.body.removeChild(link);
 }
 
+function hideParagraphCopies() {
+  const paragraphs = Array.from(document.getElementsByTagName("p")).filter(
```

Comment:
Given the calculator could be inserted anywhere on a Wordpress site eventually, we should probably be more specific with this selector so it only gets children of a `.metalation-calculator` div (in case anyone decides to give another paragraph on the page a class of `intro`).

Or I guess the easier option would just be to change the p class from `intro` to `metalation-calculator-intro`?